style.css

```diff
@@ -111,6 +111,10 @@ a code {
   top: -0.05em;
 }
 
+#search-results > div {
```

Comment:
Matching elements by tag name is inadvisable. Let's add an id or class name to the div and use a more precise selector here.